src/app/ad-authoring/ad-authoring.service.spec.ts

```diff
@@ -25,21 +25,30 @@ describe('AdAuthoringService', () => {
     expect(thisState).toEqual(state.getState());
   });
 
-  it('should update the landing page url state', () => {
+  it('should update the landing page url state and amp html should contain it', () => {
```

Comment:
We can move these into their own test cases. One test to assert the landingUrl is updated, one test to assert the AMPHTML reflects it. 

Also use AMPHTML in the test description.